### DISCUSSION TITLE
Glueton v3.38 Proton Portforwarding Not Working

### DISCUSSION THREAD

**Discussion by mikeyi99** [qdm12/gluetun#2386] [Q&A] [answered]
Just upgraded to V3.38  - Made sure the selected server was P.F. enabled, but continually seeing 'ERROR [vpn] port forwarding for the first time: server does not support port forwarding'  in logs. Please advise. Thanks!

**Comment by mlankena** [comment]
I'm running into the same thing. Are you using Wireguard? If so, I'm assuming you have your VPN_SERVICE_PROVIDER container variable set to custom? 

[This commit](https://github.com/qdm12/gluetun/commit/c5c37e7f969ee88aad8287d6679bc5f26b0ec461) added a check inside `validateFeatureFilters` which throws an error if the vpn service provider isn't PIA or Proton. Hoping one of the developers/maintainers can chime in on whether this is a configuration issue on our end, or maybe that condition needs to account for the 'VPN_PORT_FORWARDING_PROVIDER' value as well?

In the meantime, I've reverted to `pr-2285` .

**Reply by mikeyi99** [reply]
Yes, Protonvpn through WireGuard. I have the proper env variable set as you mentioned. The exact same config works flawlessly if I just change the pull version to v3.37.0

**Reply by qdm12** [reply, marked as answer]
I think that got fixed in 34e8f5f3a912d299925d293080ded245155433fb

**Reply by mikeyi99** [reply]
Yes! Thank you!

**Comment by qdm12** [comment]
@mikeyi99 

> Just upgraded to V.4

What's that if I may ask? Thanks!

**Reply by mikeyi99** [reply]
v3.38  i just saw family v4 in logs. Sorry for confusion. Latest commit seems to have resolved the issue.

**Reply by qdm12** [reply]
Ok no problem 👍 I thought Protonvpn had a v4 of something 😄
